src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const memory = {
+    result: 0,
+
+    add(value) {
+      memory.result += value;
```

Comment:
```suggestion
      this.result += value;
```

Why you don't use `this`?
The same in all other places